Teensy/src/gpio/gpio.cpp

```diff
@@ -50,7 +50,31 @@ static uint32_t gpio_getEngineRevs()
 
 static int32_t gpio_getDamperPosition()
 {
-    return 0;
+    // Defining pins //
+    #include <Arduino.h>
+
+    float exten1, exten2, exten3, exten4;
+    float DampPos1, DampPos2, DampPos3, DampPos4;
+
+    // Initialising transmission //
+    Serial.begin(9600);
+
+    // Reading the voltage outputs //
+
+    exten1 = analogRead(PIN_A0);
```

Comment:
Its good practice to avoid using the pin definitions like this. Instead at the top of this file, using an enum/define statement like the following - 
```
#define DAMPER_FRONT_RIGHT PIN_A0
```
Then use DAMPER_FRONT_RIGHT instead of PIN_A0 for analogRead()